clients/da-vinci-client/src/main/java/com/linkedin/davinci/schema/writecompute/WriteComputeHandlerV2.java

```diff
@@ -166,10 +169,20 @@ private GenericRecord assertAndGetTsRecordFieldIsRecord(GenericRecord timestampR
     if (!(fieldTimestamp instanceof GenericRecord)) {
       throw new IllegalStateException(
           String.format(
-              "Expect field %s in the timestamp record to be a generic record. Got timestamp record: %s",
-              fieldTimestamp,
+              "Expect field '%s' in the timestamp record to be a generic record. Got timestamp record: %s",
+              fieldName,
               timestampRecord));
     }
     return (GenericRecord) fieldTimestamp;
   }
+
+  // protected access for testing
+  public Object updateArray(Schema arraySchema, List originalArray, Object writeComputeArray) {
+    return super.updateArray(arraySchema, originalArray, writeComputeArray);
+  }
+
+  // protected access for testing
+  protected Object updateMap(Map originalMap, Object writeComputeMap) {
```

Comment:
Since these functions are calling `super` function directly without additional logic, so why override them at all?

Reply:
Thanks for catching this... I did this in an earlier iteration of the refactoring, but since then it's not needed anymore. I reverted this change.